src/Umbraco.Forms.Integrations.Crm.Hubspot/HubspotWorkflow.cs

```diff
@@ -69,6 +69,11 @@ public override WorkflowExecutionStatus Execute(WorkflowExecutionContext context
             }
         }
 
+        public override Task<WorkflowExecutionStatus> ExecuteAsync(WorkflowExecutionContext context)
```

Comment:
On form submit this throws an exception because the functionality resides in the incorrect `Execute` method.